src-vue/src/plugins/store.ts

```diff
@@ -313,8 +313,9 @@ export const store = createStore<FlightCoreStore>({
 
             // Display notification to highlight change
             ElNotification({
+                // TODO translate
```

Comment:
Leftover TODO? ^^

Reply:
Nah, I forgot to translate those :)
Fixed in 50161e678a96abdf233f867e5640184c15d9ee7a